scipy/special/_logsumexp.py

```diff
@@ -125,6 +141,11 @@ def logsumexp(a, axis=None, b=None, keepdims=False, return_sign=False):
     return (out, sgn) if return_sign else out
 
 
+def _wrap_radians(x):
+    # Wrap radians to -pi, pi interval
+    return (x + math.pi) % (2 * math.pi) - math.pi
```

Comment:
@steppi it occurs to me that this would lose precision near $x = 0$. The alternative, of course, is `lazywhere`. Do we want to do that?